webrender/src/device.rs

```diff
@@ -1179,7 +1213,7 @@ impl Device {
         self.init_texture(temp_texture_id, old_width, old_height, format, filter, mode, None);
         self.create_fbo_for_texture_if_necessary(temp_texture_id, None);
 
-        self.bind_render_target(Some((texture_id, 0)));
+        self.bind_render_target(Some((texture_id, 0, [1, 1])));
```

Comment:
This adds an extra glViewport call here. It should be fine, I think - but perhaps worth a comment just noting why it's ok to pass [1, 1] here.